delivery_send_to_shipper_at_operation/models/delivery_carrier.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import fields, models
+
+SEND_DELIVERY_NOTICE_ON_HELP = """
+    'Ship operation' (default): send the carrier notification after the
+    operation having a delivery method defined has been marked as done,
+    in Odoo usually the ship operation;
+
+    'Defined operations': send the carrier notification after the operation
+    of the chosen type bellow has been marked as done. In that case, the
+    carrier will be taken looking at the next operations until we found the
+    carrier (e.g. in a pick + pack + ship setup, if you set the notification
+    to be sent on the pack operation type, it'll look at the carrier defined
+    in the next operation, being here the ship).
+
+    If no notification have been sent when reaching the ship step
+    (e.g. re-route goods from carrier A to B), then a fallback ensure the
+    ship operation will trigger the call anyway.
```

Comment:
```suggestion
    'Ship operation' (default): send the carrier notification after the
    operation having a delivery method defined has been marked as done.

    'Defined operations': send the carrier notification after the operation
    of the chosen types has been marked as done. In this case, the
    carrier will be taken looking at the next operations until a
    carrier is found. For instance, in a pick + pack + ship setup,
    if you set the notification to be sent on the pack operation type,
    it will look at the carrier defined in the next operation (the ship in this case).

    If no notification have been sent when reaching the ship step
    (e.g. re-route goods from carrier A to B), then a fallback ensure the
    ship operation will trigger the call anyway.
```